packages/data-models/package.json

```diff
@@ -75,7 +75,8 @@
       "allowImportExportEverywhere": true
     },
     "rules": {
-      "no-prototype-builtins": "off"
+      "no-prototype-builtins": "off",
+      "accessor-pairs": "warn"
```

Comment:
@kirlat the accessor-pairs rule is apparently needed for inflection-tables too with the latest version of the linter. Maybe you can make the change for all of the packages?

Reply:
Sure, will update them